### DISCUSSION TITLE
Question about Qt GUI system

### DISCUSSION THREAD

**Discussion by ghost** [lxqt/lxqt#2210] [Q&A] [answered]
Hi, I was wondering if Qt5/6 use a "pixel-perfect" positioning system (wherein each "widget"s position is essentially hard-coded or related to another), or uses a tree data structure like in GTK. Thanks, I'm only vaguely familiar with GTK3. Now with GTK4 and Qt5 -> Qt6 I am rather confused.

**Comment by tsujan** [comment, marked as answer]
I didn't understand what you meant by "pixel-perfect", "hard-coded position" or "tree data structure". Qt has a comprehensive documentation. You might find answers to your questions in https://doc.qt.io/qt-6/layout.html and related pages.
